.github/workflows/integration-test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        tutor_version: ['<18.0.0', '<19.0.0', 'nightly']
+        tutor_version: ['<18.0.0', '<19.0.0']
```

Comment:
I think these changes should not reach `main` :eyes:

Reply:
@jignaciopm they shouldn't. We need to make sure to undo this before merging

Reply:
@jignaciopm can we remove this? I believe wth the latest changes the tests in nightly are not failing anymore

Reply:
@jignaciopm can we remove this? I believe with the latest changes the tests in nightly are not failing anymore